src/ucp/api/ucp.h

```diff
@@ -772,10 +775,45 @@ typedef struct ucp_ep_params {
      * 'flags' field.
      */
     ucs_sock_addr_t         sockaddr;
+
+    /**
+     * User data associated with an endpoint. See @ref ucp_ep_poll_t.
```

Comment:
ucp_ep_poll_t is not currently defined.  Will that be coming soon in another PR?

Reply:
Ahh...I think I see, it should be ucp_stream_poll_ep_t, right?

Reply:
Yes, thanks for catch!